packages/mobile/src/store/lifecycle/reducer.ts

```diff
@@ -1,12 +1,6 @@
 import type { User } from '@audius/common'
 
 import type { LifecycleActions } from './actions'
-import {
-  SIGNED_IN,
-  SIGNED_OUT,
-  ON_SIGN_UP,
-  FETCH_ACCOUNT_FAILED
-} from './actions'
 
 export type LifecycleState = {
```

Comment:
Can we just get rid of this whole reducer?

Reply:
Wasn't sure if you needed it still. will delete!

Reply:
Just need the actions for use in sagas!